src/components/AttendantDisplay.jsx

```diff
@@ -0,0 +1,14 @@
+export default function AttendantDisplay({
+  attendant: { name, lastName, jobTitle, age },
+}) {
+  return (
+    <div className="attendantsDisplay">
+      <br />
+      Name: <span>{name} </span>
+      Last Name: <span>{lastName} </span>
+      Job Title: <span>{jobTitle} </span>
+      Age: <span>{age} </span>
+      <br />
+    </div>
+  );
+}
```

Comment:
The practice is to define this as a `const SomeComponent` and then export it as default at the bottom of the file.
Also the name can be simplified.
```suggestion
const Attendant = ({
  attendant: { name, lastName, jobTitle, age },
}) => (
    <div className="attendantsDisplay">
      <br />
      Name: <span>{name} </span>
      Last Name: <span>{lastName} </span>
      Job Title: <span>{jobTitle} </span>
      Age: <span>{age} </span>
      <br />
    </div>
  );
  
  export default Attendant
```